modules/omx/omx.c

```diff
@@ -251,28 +251,37 @@ int omx_display_enable(struct omx_state* st,
 	/* specify buffer requirements */
 	err |= OMX_GetParameter(st->video_render,
 		OMX_IndexParamPortDefinition, &portdef);
+	if (err != 0) {
+		error("omx_display_enable: couldn't retrieve port def\n");
+		err = ENOMEM;
+		goto exit;
+	}
+
+	info("omx port definition: h=%d w=%d s=%d sh=$d\n",
+		portdef.format.video.nFrameWidth,
+		portdef.format.video.nFrameHeight,
+		portdef.format.video.nStride,
+		portdef.format.video.nSliceHeight);
```

Comment:
the forth format specified says `$d` is that on purpose?
should it perhaps be `%d` instead?